build_tools/azure/py38_conda_defaults_openblas_environment.yml

```diff
@@ -0,0 +1,20 @@
+channels:
+  - defaults
+dependencies:
+  - python=3.8
+  - numpy=1.17.3  # min
+  - blas[build=openblas]
+  - scipy=1.3.2  # min
+  - cython
+  - joblib
+  - threadpoolctl=2.2.0
```

Comment:
Not sure why we pinned this one. Maybe we could set it to `# min` as well unless it breaks the build and in which case we can explain why in a comment.

But maybe we can do that in a later PR.

Reply:
I tried to mirror what was done in main, but I agree there are some weird quirks here and there, probably best left to a further PR.